.golangci.yaml

```diff
@@ -1,6 +0,0 @@
-issues:
-  exclude-rules:
-    - linters:
-        - staticcheck
-      text: "SA1019:"
-
```

Comment:
This was introduced in #207 due to the linters were unhappy with the deprecation warnings in some of the files and workaround for that was just disable it. Since this PR uplifting CAPI, we do not need this file and linter can live without it.

Reply:
Ok